demo/src/style.css

```diff
@@ -116,7 +116,291 @@ path.nad-arrow-in:not(.nad-state-in .nad-arrow-in) {visibility: hidden}
+    .nad-text-edges,
+    [class^="nad-vl0to30"],
+    [class*=" nad-vl0to30"],
+    [class^="nad-vl30to50"],
+    [class*=" nad-vl30to50"],
+    [class^="nad-vl50to70"],
+    [class*=" nad-vl50to70"],
+    [class^="nad-vl70to120"],
+    [class*=" nad-vl70to120"],
+    [class^="nad-vl120to180"],
+    [class*=" nad-vl120to180"],
+    [class^="nad-vl180to300"],
+    [class*=" nad-vl180to300"] {
+        display: block
+    }
+    .nad-disconnected .nad-edge-path {
+        stroke-dasharray: 10 10
+    }
+}
+
+.nad-zoom-1000 {
+
+    .nad-branch-edges .nad-edge-path,
+    .nad-3wt-edges .nad-edge-path,
+    .nad-branch-edges .nad-winding,
+    .nad-3wt-nodes .nad-winding,
+    .nad-vl-nodes circle.nad-unknown-busnode {
+        stroke-width: 0.25%
+    }
+    .nad-edge-infos,
+    .nad-label-box,
+    .nad-text-edges,
+    [class^="nad-vl0to30"],
+    [class*=" nad-vl0to30"],
+    [class^="nad-vl30to50"],
+    [class*=" nad-vl30to50"],
+    [class^="nad-vl50to70"],
+    [class*=" nad-vl50to70"],
+    [class^="nad-vl70to120"],
+    [class*=" nad-vl70to120"],
+    [class^="nad-vl120to180"],
+    [class*=" nad-vl120to180"],
+    [class^="nad-vl180to300"],
+    [class*=" nad-vl180to300"] {
+        display: block
+    }
+    .nad-disconnected .nad-edge-path {
+        stroke-dasharray: 10 10
+    }
+}
+.nad-zoom-2200 {
+    .nad-branch-edges .nad-edge-path,
+    .nad-3wt-edges .nad-edge-path,
+    .nad-branch-edges .nad-winding,
+    .nad-3wt-nodes .nad-winding,
+    .nad-vl-nodes circle.nad-unknown-busnode {
+        stroke-width: 0.25%
+    }
+    .nad-edge-infos {
+        display: none
+    }
+    .nad-label-box,
+    .nad-text-edges,
+    [class^="nad-vl0to30"],
+    [class*=" nad-vl0to30"],
+    [class^="nad-vl30to50"],
+    [class*=" nad-vl30to50"],
+    [class^="nad-vl50to70"],
+    [class*=" nad-vl50to70"],
+    [class^="nad-vl70to120"],
+    [class*=" nad-vl70to120"],
+    [class^="nad-vl120to180"],
+    [class*=" nad-vl120to180"],
+    [class^="nad-vl180to300"],
+    [class*=" nad-vl180to300"] {
+        display: block
+    }
+    .nad-disconnected .nad-edge-path {
+        stroke-dasharray: 10 10
+    }
+}
+.nad-zoom-2500 {
+    .nad-branch-edges .nad-edge-path,
+    .nad-3wt-edges .nad-edge-path,
+    .nad-branch-edges .nad-winding,
+    .nad-3wt-nodes .nad-winding,
+    .nad-vl-nodes circle.nad-unknown-busnode {
+        stroke-width: 0.25%
+    }
+    .nad-edge-infos {
+        display: none
+    }
+    .nad-label-box,
+    .nad-text-edges,
+    [class^="nad-vl0to30"],
+    [class*=" nad-vl0to30"],
+    [class^="nad-vl30to50"],
+    [class*=" nad-vl30to50"],
+    [class^="nad-vl50to70"],
+    [class*=" nad-vl50to70"],
+    [class^="nad-vl70to120"],
+    [class*=" nad-vl70to120"],
+    [class^="nad-vl120to180"],
+    [class*=" nad-vl120to180"],
+    [class^="nad-vl180to300"],
+    [class*=" nad-vl180to300"] {
+        display: block
+    }
+    .nad-disconnected .nad-edge-path {
+        stroke-dasharray: 0.5%, 0.5%
+    }
```

Comment:
I don't see the point to define this in percentage, but maybe I'm missing something? Could you remove anything expressed in percentage? The svg has been drawn in powsybl-diagram for some values which might be changed a bit, but changing that much the dimensions is spoiling the this makes the diagram quite unattractive.

Reply:
I guess percentage was used to allow for smooth transition without defining many zoom levels css classes only to change line size. We can discuss this.

Reply:
Ok, we could put this in a separate PR, so that we first keep the same behaviour with this change